### DISCUSSION TITLE
Nova doesnt use asset() helper for assets

### DISCUSSION THREAD

**Discussion by breadlesscode** [laravel/nova-issues#3249] [Q&A] [answered]
- Laravel Version: v8.28.1
- Nova Version: 3.22.0
- PHP Version: 7.4.14
- Database Driver & Version: MySQL 8.0.17
- Operating System and Version: Mac 
- Browser type and version: ALL
- Reproduction Repository: ./resources/views/layout.blade.php Line 79-81

### Description:
We are servering our assets from a subdirectory, which works fine with the `asset()` helper. But Laravel Nova doesnt use `asset()` for the asset urls. 

### Detailed steps to reproduce the issue on a fresh Nova installation:
Define `ASSET_URL` in .env and se that its doenst change the URLs in Nova backend. 

Can we integrate the `asset()` helper in the Nova? 

Awesome project btw :) <3

**Comment by crynobone** [comment, marked as answer]
You should use `Nova::remoteScript()` if you have this requirement.
